src/Game.java

```diff
@@ -0,0 +1,116 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public class Game {
+    private final Board board;
+    private final Map<Integer,Integer> snakeMap;
+    private final Map<Integer,Integer> ladderMap;
+    private boolean gameWon;
+
+    public Game(Board board) {
+        this.board = board;
+        this.snakeMap = new HashMap<>();
+        this.ladderMap = new HashMap<>();
+        this.gameWon = false;
+    }
+
+    public Board getBoard() {
+        return this.board;
+    }
+
+    public boolean getGameWon() {
+        return this.gameWon;
+    }
+
+    public void setGameWon() {
+        this.gameWon = true;
+    }
+
+    public void addSnakeToBoard(int startCoordinate, int endCoordinate) {
+        this.snakeMap.put(startCoordinate, endCoordinate);
+    }
+
+    public void addLadderToBoard(int startCoordinate, int endCoordinate) {
+        this.ladderMap.put(startCoordinate, endCoordinate);
+    }
+
+    /*public boolean isGameWon() {
+        for(Player player : this.board.getPlayerList()) {
+            if(player.getHasWon())
+                return true;
+        }
+        return false;
+    }*/
+
+    private void printMove(Player player, int diceNumber, int start, int end) {
+        System.out.println(player.getPlayerName() + " rolled a " + diceNumber + " and moved from " +
+                           start + " to " + end);
+        return;
+    }
+
+    private int updateDieNumber() {
+        int val1 = 6;
```

Comment:
use proper variable names

Reply:
I agree, the variable names could be made better.